test/junit/scala/collection/IterableTest.scala

```diff
@@ -55,124 +56,116 @@ class IterableTest {
-    assert(Map.empty      == immutable.Map.empty.toTraversable)
+    assert(Map.empty == immutable.ListMap.empty.toTraversable)
+    assert(Set.empty == immutable.ListSet.empty.toTraversable)
+    assert(Map.empty == immutable.LongMap.empty.toTraversable)
+    assert((0 to 0) == ((0 to 0): immutable.Range.Inclusive).toTraversable)
+    assert((0 until 1) == ((0 until 1): immutable.Range).toTraversable)
+    assert(Map.empty == immutable.Map.empty.toTraversable)
     assert(Iterable.empty == immutable.Seq.empty.toTraversable)
-    assert(Set.empty      == immutable.Set.empty.toTraversable)
+    assert(Set.empty == immutable.Set.empty.toTraversable)
     assert(Iterable.empty == immutable.Stream.empty.toTraversable)
     assert(Iterable.empty == ("": String).toTraversable)
     assert(Iterable.empty == immutable.Traversable.empty.toTraversable)
     assert(Iterable.empty == immutable.Vector.empty.toTraversable)
     assert(Iterable.empty == mutable.ArrayBuffer.empty.toTraversable)
     assert(Iterable.empty == mutable.ArraySeq.empty.toTraversable)
     assert(Iterable.empty == mutable.ArrayStack.empty.toTraversable)
-    assert(Set.empty      == mutable.BitSet.empty.toTraversable)
+    assert(Set.empty == mutable.BitSet.empty.toTraversable)
     assert(Iterable.empty == mutable.Buffer.empty.toTraversable)
-    assert(Map.empty      == mutable.HashMap.empty.toTraversable)
-    assert(Set.empty      == mutable.HashSet.empty.toTraversable)
+    assert(Map.empty == mutable.HashMap.empty.toTraversable)
+    assert(Set.empty == mutable.HashSet.empty.toTraversable)
     assert(Iterable.empty == mutable.IndexedSeq.empty.toTraversable)
     assert(Iterable.empty == mutable.Iterable.empty.toTraversable)
-    assert(Map.empty      == mutable.ListMap.empty.toTraversable)
-    assert(Map.empty      == mutable.LongMap.empty.toTraversable)
-    assert(Map.empty      == mutable.Map.empty.toTraversable)
+    assert(Map.empty == mutable.ListMap.empty.toTraversable)
+    assert(Map.empty == mutable.LongMap.empty.toTraversable)
+    assert(Map.empty == mutable.Map.empty.toTraversable)
     assert(Iterable.empty == mutable.Queue.empty.toTraversable)
     assert(Iterable.empty == mutable.Seq.empty.toTraversable)
     assert(Iterable.empty == mutable.Stack.empty.toTraversable)
     assert(Iterable.empty == mutable.Traversable.empty.toTraversable)
-    assert(Map.empty      == mutable.LinkedHashMap.empty.toTraversable)
-    assert(Set.empty      == mutable.LinkedHashSet.empty.toTraversable)
+    assert(Map.empty == mutable.LinkedHashMap.empty.toTraversable)
+    assert(Set.empty == mutable.LinkedHashSet.empty.toTraversable)
     assert(Iterable.empty == mutable.ListBuffer.empty.toTraversable)
-    assert(Map.empty      == mutable.OpenHashMap.empty.toTraversable)
-    assert(Set.empty      == mutable.Set.empty.toTraversable)
+    assert(Map.empty == mutable.OpenHashMap.empty.toTraversable)
+    assert(Set.empty == mutable.Set.empty.toTraversable)
```

Comment:
This is a little sad.. 😕

Reply:
+1, I would like to keep the alignment

Reply:
:+1: will fix

Reply:
@dwijnand @NthPortal btw this is fixed now